dotCMS/src/main/webapp/html/portlet/ext/workflows/schemes/workflow_main.jsp

```diff
@@ -8,7 +8,7 @@
 	@import "/html/js/dragula-3.7.2/dragula.min.css"; 
 </style>	
 
-<div class="portlet-wrapper" <%if(LicenseUtil.getLevel() < LicenseLevel.STANDARD.level){ %>style="background:#F4F3F4 url(/html/images/skin/workflow-background.png) repeat-x 0 0 " <%} %>>
+<div class="portlet-wrapper">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5587482)